src/layout/layout.js

```diff
@@ -401,13 +401,15 @@ function MaterialLayoutTab(tab, tabs, panels, layout) {
     }
 
     tab.addEventListener('click', function(e) {
-      e.preventDefault();
-      var href = tab.href.split('#')[1];
-      var panel = layout.content_.querySelector('#' + href);
-      layout.resetTabState_(tabs);
-      layout.resetPanelState_(panels);
-      tab.classList.add(layout.CssClasses_.IS_ACTIVE);
-      panel.classList.add(layout.CssClasses_.IS_ACTIVE);
+      if (tab.getAttribute('href').charAt(0) === '#') {
```

Comment:
Honest question: Wouldn’t an anchor also work if the page is referenced explicitly?

Or to phrase it the other way around: If I am on `index.html`, will `index.html#someAnchor` trigger a reload? This case should probably be handled as well as some static site generator setups might enforce absolute links.